src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java

```diff
@@ -6,13 +6,15 @@
     private String userEmail;
     private String login;
     private String password;
+    private int amountOfDays;
```

Comment:
The `amountOfDays` is not a part of `AuthData`. Please remove it out of here.

Reply:
Done